go.mod

```diff
@@ -0,0 +1,89 @@
+	github.com/stretchr/testify v1.3.0
+	github.com/stvp/rollbar v0.5.1
+	github.com/twmb/algoimpl v0.0.0-20170717182524-076353e90b94
+	golang.org/x/crypto v0.0.0-20190228161510-8dd112bcdc25
+	gopkg.in/cheggaaa/pb.v1 v1.0.28
+	gopkg.in/yaml.v2 v2.2.2
+	k8s.io/api v0.0.0-20180628040859-072894a440bd
+	k8s.io/apimachinery v0.0.0-20180621070125-103fd098999d
+	k8s.io/client-go v8.0.0+incompatible
+	k8s.io/metrics v0.0.0-20180628054111-6f051017e10b
+)
+
+require (
+	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
+	github.com/Microsoft/go-winio v0.4.5 // indirect
+	github.com/Sirupsen/logrus v1.0.3 // indirect
+	github.com/andybalholm/cascadia v0.0.0-20161224141413-349dd0209470 // indirect
+	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 // indirect
+	github.com/convox/inotify v0.0.0-20170313035821-b56f5149b5c6 // indirect
+	github.com/crazy-max/xgo v0.7.4 // indirect
+	github.com/docker/spdystream v0.0.0-20170912183627-bc6354cbbc29 // indirect
+	github.com/elazarl/goproxy v0.0.0-20210801061803-8e322dfb79c4 // indirect
+	github.com/ghodss/yaml v1.0.0 // indirect
+	github.com/gogo/protobuf v1.1.1 // indirect
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b // indirect
+	github.com/golang/groupcache v0.0.0-20180513044358-24b0969c4cb7 // indirect
+	github.com/google/btree v1.0.0 // indirect
+	github.com/google/gofuzz v0.0.0-20170612174753-24818f796faf // indirect
+	github.com/googleapis/gnostic v0.2.0 // indirect
+	github.com/gregjones/httpcache v0.0.0-20180305231024-9cad4c3443a7 // indirect
+	github.com/hashicorp/golang-lru v0.0.0-20180201235237-0fb14efe8c47 // indirect
+	github.com/headzoo/ut v0.0.0-20181013193318-a13b5a7a02ca // indirect
+	github.com/jehiah/go-strftime v0.0.0-20171201141054-1d33003b3869 // indirect
+	github.com/json-iterator/go v1.1.5 // indirect
+	github.com/karalabe/xgo v0.0.0-20191115072854-c5ccff8648a7 // indirect
+	github.com/mattn/go-runewidth v0.0.4 // indirect
+	github.com/moby/moby v1.13.1 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742 // indirect
+	github.com/opencontainers/runc v0.1.1 // indirect
+	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
+	github.com/segmentio/backo-go v0.0.0-20160424052352-204274ad699c // indirect
+	github.com/xtgo/uuid v0.0.0-20140804021211-a0b114877d4c // indirect
+	golang.org/x/net v0.0.0-20190213061140-3a22650c66bd // indirect
+	golang.org/x/sys v0.0.0-20190302025703-b6889370fb10 // indirect
+	golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2 // indirect
+	gopkg.in/inf.v0 v0.9.1 // indirect
+	k8s.io/kube-openapi v0.0.0-20180731170545-e3762e86a74c // indirect
+)
+
+replace gopkg.in/yaml.v2 => github.com/ddollar/yaml v0.0.0-20180504010936-3fb95e32dd8a
+
+replace github.com/fsouza/go-dockerclient => github.com/heronrs/go-dockerclient v0.0.1
```

Comment:
The current `go-dockerclient` version we use is outdated and incompatible with `go.mod`. Updating it to a later version unchained a very complex cascade of outdated dependencies(including packages that changed their name such as docker/moby and case-insensitive name collisions (Sirupsen/sirupsen on the logrus package)).

So I just forked the repo and made the version we use work with go modules.